ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/schema/WithdrawRequestV1.java

```diff
@@ -32,10 +35,17 @@ public class ExitV1 {
   @JsonDeserialize(using = Bytes48Deserializer.class)
   public final Bytes48 validatorPublicKey;
 
-  public ExitV1(
+  @JsonSerialize(using = UInt64AsHexSerializer.class)
+  @JsonDeserialize(using = UInt64AsHexDeserializer.class)
+  public final UInt64 amount;
+
+  public WithdrawRequestV1(
       @JsonProperty("sourceAddress") final Bytes20 sourceAddress,
-      @JsonProperty("validatorPublicKey") final Bytes48 validatorPublicKey) {
+      /* TODO(JWT): shouldn't this be called validatorPubkey? */
```

Comment:
Engine API spec has it as `validatorPublicKey`

https://github.com/ethereum/execution-apis/blob/main/src/engine/prague.md#exitv1

Reply:
schema objects we don't use really, so its probably not a huge problem, more the SSZ object thats going to be important...